xonsh/commands_cache.py

```diff
@@ -126,10 +125,12 @@ def lazy_locate_binary(self, name):
         if ON_WINDOWS:
             # Windows users expect to be able to execute files in the same
             # directory without `./`
-            local_bin = next((fn for fn in possibilities if os.path.isfile(fn)), None)
+            local_bin = next((fn for fn in possibilities if os.path.isfile(fn)),
+                             None)
             if local_bin:
-                return os.path.abspath(local_bin)                              
-        cached = next((cmd for cmd in possibilities if cmd in self._cmds_cache), None)
+                return os.path.abspath(local_bin)
+        cached = next((cmd for cmd in possibilities if cmd in self._cmds_cache),
+                      None)
         if cached:
```

Comment:
Oh no.  Was I just PEP8'ed here? Can we change the line endings rules so it doesn't complain hard if we go a few characters above 80. That rule is not giving nicer code ...